modules/update-upgrading-web.adoc

```diff
@@ -47,9 +47,9 @@ such as `stable-{product-version}`.
 ====
 For production clusters, you must subscribe to a stable-* or fast-* channel.
 ====
-** If the *UPDATE STATUS* is not *Updates Available*, you cannot upgrade your
+** If the *Update Status* is not *Updates Available*, you cannot upgrade your
 cluster.
-** The *DESIRED VERSION* indicates the cluster version that your cluster is running
+** *Select Channel* indicates the cluster version that your cluster is running
 or is updating to.
 
 . Click *Updates Available*, select
```

Comment:
For 4.6, to choose a version to update, need click the button "Update". No button "Updates Available"